controller.go

```diff
@@ -357,6 +357,10 @@ func (c *controller) NewNetwork(networkType, name string, options ...NetworkOpti
 		return nil, err
 	}
 
+	if network.Type() == "bridge" && network.isolated {
+		return nil, ErrNetworkNotSupportIsolation(network.Type())
```

Comment:
So I should define an `isolated` label and pass isolation option to `n.generic` before https://github.com/docker/libnetwork/blob/master/controller.go#L407 ?